core/src/main/resources/templates/core/management_events.html

```diff
@@ -355,29 +384,29 @@ <h4 class="modal-title">[[#{greenCity.events.page.edit.events}]]</h4>
             </div>
         </div>
         <!-- Delete Modal HTML -->
-        <div id="deleteEventsModal" class="modal fade">
-            <div class="modal-dialog">
-                <div class="modal-content">
-                    <form id="deleteEventsForm">
-                        <div class="modal-header">
-                            <h4 class="modal-title">[[#{greenCity.events.page.delete.events}]]</h4>
-                            <button type="button" class="close" data-dismiss="modal" aria-hidden="true">&times;</button>
-                        </div>
-                        <div class="modal-body">
-                            <p>[[#{greenCity.pages.popup.delete.h2}]]</p>
-                            <p class="text-warning"><small>[[#{greenCity.pages.popup.delete.all.h2}]]</small></p>
-                        </div>
-                        <div class="modal-footer">
-                            <input type="button" class="btn btn-default btn-tertiary" data-dismiss="modal"
-                                   th:value="#{greenCity.pages.popup.cancel.}">
-                            <a href="" id="deleteOneSubmit" class="btn btn-remove">
-                                <div>[[#{greenCity.pages.delete}]]</div>
-                            </a>
-                        </div>
-                    </form>
-                </div>
-            </div>
-        </div>
+<!--        <div id="deleteEventsModal" class="modal fade">-->
+<!--            <div class="modal-dialog">-->
+<!--                <div class="modal-content">-->
+<!--                    <form id="deleteEventsForm">-->
+<!--                        <div class="modal-header">-->
+<!--                            <h4 class="modal-title">[[#{greenCity.events.page.delete.events}]]</h4>-->
+<!--                            <button type="button" class="close" data-dismiss="modal" aria-hidden="true">&times;</button>-->
+<!--                        </div>-->
+<!--                        <div class="modal-body">-->
+<!--                            <p>[[#{greenCity.pages.popup.delete.h2}]]</p>-->
+<!--                            <p class="text-warning"><small>[[#{greenCity.pages.popup.delete.all.h2}]]</small></p>-->
+<!--                        </div>-->
+<!--                        <div class="modal-footer">-->
+<!--                            <input type="button" class="btn btn-default btn-tertiary" data-dismiss="modal"-->
+<!--                                   th:value="#{greenCity.pages.popup.cancel.}">-->
+<!--                            <a href="" id="deleteOneSubmit" class="btn btn-remove">-->
+<!--                                <div>[[#{greenCity.pages.delete}]]</div>-->
+<!--                            </a>-->
+<!--                        </div>-->
+<!--                    </form>-->
+<!--                </div>-->
+<!--            </div>-->
+<!--        </div>-->
```

Comment:
Did you comment that out on purpose? Either delete it completely or bring this back